### ISSUE TITLE
problem with my smapi

### ISSUE THREAD

**Issue by Paradiis** [ZaneYork/SMAPI-Android-Installer#486]
![Screenshot_20230714-165015767](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/137906640/24721f7b-4ae3-4368-ae50-b09f5e6f1dfa)

I have a problem where, when I'm in the "SMAPI need to locate stardew valley folder" section, when I press ok, it suddenly exits the game. I tried to do it manually by moving the smapi file outside of me move it to the android file "com.zane.stardewvalley" but red text appears in the console section. Please help and find a solution

**Comment by RUNNEARS** [comment]
Same problem. My phone is xiaomi redmi 9T
I need solution.

**Comment by RUNNEARS** [comment]
![Screenshot_2023-07-15-05-08-54-779_com zane stardewvalley](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/119922628/c87f4d16-1617-4b6a-9e56-040461a04110)
Same problem. My phone is xiaomi redmi 9T
I need solution.

**Comment by CoderMonkey1956** [comment]
Same issue also, made a ticket about it here https://github.com/ZaneYork/SMAPI-Android-Installer/issues/398 but even that com.zane.starduevalley trick is blocked by Android restrictions, I kinda found an exploit using some kinda Google shell bug but even then the folder dossent show up so you can't do the workaround. Hope someone is able to help with this.

**Comment by CoderMonkey1956** [comment]
> ![Screenshot_20230714-165015767](https://user-images.githubusercontent.com/137906640/253538246-24721f7b-4ae3-4368-ae50-b09f5e6f1dfa.jpg)
> 
> I have a problem where, when I'm in the "SMAPI need to locate stardew valley folder" section, when I press ok, it suddenly exits the game. I tried to do it manually by moving the smapi file outside of me move it to the android file "com.zane.stardewvalley" but red text appears in the console section. Please help and find a solution

Hey Everyone! I found a solution to the issue! First, open up a file manager that can view your most recent files and view them right after installing SMAPI and getting the error. Then you'll see a lot of .dll .p7b and .json files made by SMAPI, delete them all... then reinstall SMAPI (and the installer) and it should work! Hope this helps!

**Comment by redblueatoms** [comment]
I got the file in android/data by connecting to a PC and manually unzipping it then moving it to the phone but now my pathing is broken and causing crashes I've tried manually typing and copy pasting the location nothing works 
![Screenshot_20230721-205458](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/140227536/c782363a-3cf0-4517-9752-fda007aad98e)

**Comment by CoderMonkey1956** [comment]
> > ![Screenshot_20230714-165015767](https://user-images.githubusercontent.com/137906640/253538246-24721f7b-4ae3-4368-ae50-b09f5e6f1dfa.jpg)
> > I have a problem where, when I'm in the "SMAPI need to locate stardew valley folder" section, when I press ok, it suddenly exits the game. I tried to do it manually by moving the smapi file outside of me move it to the android file "com.zane.stardewvalley" but red text appears in the console section. Please help and find a solution
> 
> Hey Everyone! I found a solution to the issue! First, open up a file manager that can view your most recent files and view them right after installing SMAPI and getting the error. Then you'll see a lot of .dll .p7b and .json files made by SMAPI, delete them all... then reinstall SMAPI (and the installer) and it should work! Hope this helps!



> I got the file in android/data by connecting to a PC and manually unzipping it then moving it to the phone but now my pathing is broken and causing crashes I've tried manually typing and copy pasting the location nothing works ![Screenshot_20230721-205458](https://user-images.githubusercontent.com/140227536/255312343-c782363a-3cf0-4517-9752-fda007aad98e.png)

Did you try deleting old files? Not just the app.

**Comment by redblueatoms** [comment]
Yes I have tried both of these suggestions and now I repeatedly get chose
the file location over and over again and it doesn't show the log or
install of any of my apps just keeps opening base game and saying it's
modded I don't have any text to show. Nothing with initialize.

**Comment by redblueatoms** [comment]
I can't even get mine to initialize any apps now it just opens base game you start a save and it crashes
